src/Java.Interop/Java.Interop/JniRuntime.JniMarshalMemberBuilder.cs

```diff
@@ -87,7 +87,7 @@ public  Delegate                                                CreateMarshalToM
 			{
 				if (value == null)
 					throw new ArgumentNullException (nameof (value));
-				return CreateMarshalToManagedExpression (value.GetMethodInfo ()).Compile ();
+				return CreateMarshalToManagedExpression (value.GetMethodInfo ()!).Compile ();
```

Comment:
`.GetMethodInfo()` can return null?!

Are the NRT annotations in `mscorlib.dll` "final" (or "sane")?

Reply:
Looks like this was fixed in a later version, I'll take the change out and assume we will eventually be on the same version as dotnet: https://github.com/dotnet/runtime/commit/505341df91d452f281dadc4f2638bf49d5fd0fdb.